FrontMen/fm-hours
# Warning if week or daily expected hours exceeded

# Changes
A prompt to double-check the set hours are as desired. 

## Related issues
closes https://github.com/FrontMen/fm-hours/issues/4

## Changed
Totals calculated on weekly-timesheet used in records.
When submitting for approval, based on totals, throw a confirmation prompt to check if numbers are correct.

## How to test
1. Go to timesheet
2. Fill one day with more hours than expected
or
2. Fill the week exceeding expected hours (like throw in a 9 on one day, or multiple project of 8 hours a day each) 

3. Hit submit for approval
4. See prompt - test cancel / ok correctly perform as expected

The prompt is different for week and day warning. If week is exceeded, it shouldn't show the day prompt.

## Screenshots
![image](https://user-images.githubusercontent.com/85111889/127191539-855e88ac-d1a7-4353-8fba-9182dfa76988.png)
![image](https://user-images.githubusercontent.com/85111889/127191555-0361bba7-4479-457c-b073-72f28ec4f607.png)
